tests/execution_engine/test_sqlalchemy_execution_engine.py

```diff
@@ -1083,7 +1083,7 @@ def test_execute_query(self, sa, pd_dataframe: pd.DataFrame):
             pd_dataframe, sa
         )
 
-        select_all = "SELECT * FROM test;"
+        select_all = f"SELECT * FROM {sa.text('test')};"
         result = execution_engine.execute_query(sa.text(select_all)).fetchall()
```

Comment:
@alexsherstinsky since we convert the `select_all` statement to a `sa.text()` in the following line, is this line redundant?

Reply:
@Shinnnyshinshin Apparently, what the checker wanted was strict syntax of enclosing textual table names into `sa.text()` -- that was the error message I got.  So, it sounds that wrapping `sa.text()` around `select_all` was not quite formal enough for it.  But now, all tests pass.  Thanks.